app/views/browse/config-maps.html

```diff
@@ -10,7 +10,13 @@
             <div class="pull-right" ng-if="project && ('configmaps' | canI : 'create')">
               <a ng-href="project/{{project.metadata.name}}/create-config-map" class="btn btn-default">Create Config Map</a>
             </div>
-            <h1>Config Maps</h1>
+            <h1>Config Maps
```

Comment:
Formatting nit: When there are other elements inside the header, I find it much easier to read if the header text is on the next line...

```html
<h1>
  Config Maps
  <span>...</span>
</h1>
```